home/templates/professor.html

```diff
@@ -210,7 +219,8 @@ <h5>Average rating: <span id="professor_rating">${average_rating.toFixed(2)}</sp
 		$("#professor-stats").html(stats);
 	}
 
-	function unverifyReview(form) {
+	function reviewAction(form, verified_status) {
+		$(`${form} input[name="verified"]`).val(verified_status);
```

Comment:
I'm confused, what is this line:

```
$(`${form} input[name="verified"]`).val(verified_status);
```

doing? Why does the form's `verified` input need to be changed?

Reply:
I accidentally deleted my original response while trying to remove the review I had started but it doesn't matter anyway because these lines don't exists anymore